lib/entities/scheduled-action.ts

```diff
@@ -33,13 +33,17 @@ type SchedulableActionType = 'publish' | 'unpublish'
 export type ScheduledActionSysProps = {
   id: string
   type: 'ScheduledAction'
+  version?: number
   space: { sys: MetaLinkProps }
   status: ScheduledActionStatus
   createdAt: ISO8601Timestamp
   createdBy: { sys: MetaLinkProps }
   /** an ISO8601 date string representing when an action was moved to canceled */
   canceledAt?: ISO8601Timestamp
   canceledBy?: { sys: MetaLinkProps }
+  /** an ISO8601 date string representing when an action was updated */
+  updatedAt: ISO8601Timestamp
+  updatedBy: { sys: MetaLinkProps }
```

Comment:
I added a `Link` type here, maybe we could use  `updatedBy: Link<'User'>` instead to make it more clear